core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs

```diff
@@ -260,6 +263,7 @@ fn new_confirm_output_obj(_args: &[Obj], kwargs: &Map) -> Result<Obj, error::Err
         .with_footer(TR::instructions__swipe_up.into(), None)
         .with_chunkify(chunkify)
         .with_text_mono(text_mono)
+        .with_swipe_up()
```

Comment:
This was missing? :thinking:

Reply:
It defaulted to `true` before, but I made it default to `false` so I had to add it explicitly to the places where it was actually needed.

It is actually unrelated to the issue at hand, but I noticed that if this is `true` on the screen where address is confirmed, you can swipe up and the address gets our of the screen and you are stuck with a strange empty screen... Which is why I made it `false` in that case.